GPSTest/build.gradle

```diff
@@ -126,18 +126,18 @@ tasks.whenTaskAdded { theTask ->
 }
 
 dependencies {
-    implementation 'com.android.support:appcompat-v7:27.1.1'
-    implementation 'com.android.support:design:27.1.1'
-    implementation 'com.android.support:recyclerview-v7:27.1.1"'
-    implementation 'com.android.support:cardview-v7:27.1.1'
+    implementation 'com.android.support:appcompat-v7:28.0.0-alpha3'
```

Comment:
Are non-alpha releases out yet for compileSdkVersion 28?  If not, please revert back to 27.